src/App.jsx

```diff
@@ -4,6 +4,7 @@ import { Alert } from "./components/Alert/alert";
 import { useAlert } from "./context/alert-context";
 import { Router } from "./router/Router";
 import { useAuth } from "./context/auth-context";
+import { useTheme } from "./context/theme-context";
```

Comment:
Suggestion: can make `index.js` file in context folder and export all context file from there

Reply:
will do thanks!